tests/test_plotting.py

```diff
@@ -35,404 +35,564 @@
+        return fig
 
-@pytest.mark.slow
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_plot_field_line(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
-    fig, ax, _ = plot_field_lines_sfl(eq, rho=1, seed_thetas=0, phi_end=2 * np.pi)
-    return fig
-
-
-@pytest.mark.slow
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_3d)
-def test_plot_field_lines(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
-    fig, ax, _ = plot_field_lines_sfl(
-        eq, rho=1, seed_thetas=np.linspace(0, 2 * np.pi, 4), phi_end=2 * np.pi
-    )
-    return fig
+    @pytest.mark.unit
+    @pytest.mark.solve
+    @pytest.mark.slow
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_3d)
+    def test_plot_field_lines(self, DSHAPE_current):
+        """Test plotting multiple field lines over 1 transit."""
+        eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
+        fig, ax, _ = plot_field_lines_sfl(
+            eq, rho=1, seed_thetas=np.linspace(0, 2 * np.pi, 4), phi_end=2 * np.pi
+        )
+        return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_plot_boozer_modes(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_plot_boozer_modes(DSHAPE_current):
+    """Test plotting boozer spectrum."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_boozer_modes(eq)
     ax.set_ylim([1e-12, 1e0])
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_plot_boozer_surface(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_plot_boozer_surface(DSHAPE_current):
+    """Test plotting B in boozer coordinates."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_boozer_surface(eq, figsize=(4, 4))
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_plot_qs_error(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_plot_qs_error(DSHAPE_current):
+    """Test plotting qs error metrics."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_qs_error(eq, helicity=(0, 0), log=False)
     return fig
 
 
+# @pytest.mark.unit
+# @pytest.mark.solve
+# @pytest.mark.mpl_image_compare(tolerance=50)
+# def test_plot_qh_optimization(precise_QH):
+#     """Test plotting B in boozer coordinates from precise QH optimization."""
+#     eq = EquilibriaFamily.load(load_from=str(precise_QH["optimal_h5_path"]))
+#     fig, ax = plot_boozer_surface(eq)
+#     return fig
```

Comment:
Up to you. Does this exercise code that isn't otherwise tested, or verify properties of the precise qh optimization we can't get with the other tests (I believe they just check that the non-axisymmetric Boozer modes are small). 
If we want to keep it it's probably best to combine it with the `test_precise_QH_optimization3` test.

Reply:
this is being added back as part of the `test_precise_QH_optimization3` test, just need to regenerate the baseline image.